demo/showcase/ansys-shapeopt/sdf_fd_tess/tesseract_requirements.txt

```diff
@@ -0,0 +1,6 @@
+numpy==1.26.4
+pyvista==0.45.2
+manifold3d==3.2.1
+trimesh==4.9.0
+pysdf==0.1.9
+tesseract-core==1.1.1
```

Comment:
is it intentional to include tesseract-core as a dependency inside a tesseract? why would that be needed?